extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -192,11 +194,15 @@ define('ext.wikia.adEngine.adEngine', [
 			decorators.push(adDecoratorLegacyParamFormat);
 		}
 
+		log(['run', 'adding adInfo listener'], 'debug', logGroup);
+		if (adInfoTracker && adInfoTracker.run) { //TODO: remove if check in 2017
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)